spec/models/participant_profile_spec.rb

```diff
@@ -13,9 +14,10 @@
 
   it "updates the updated_at on the users" do
     freeze_time
-    user = create(:user, :mentor, updated_at: 2.weeks.ago)
+    user = create(:user, updated_at: 2.weeks.ago)
+    profile = create(:participant_profile, user: user)
 
-    user.mentor_profile.touch
+    profile.touch
     expect(user.reload.updated_at).to be_within(1.second).of Time.zone.now
   end
```

Comment:
surprised this one works without the touch on user in teacher profile?

Reply:
Indeed, will investiagate :)